RELEASE.md

```diff
@@ -0,0 +1,75 @@
+# Release management for protocol buffer specifications
+
+This repository primary provides two features:
+
+* Protobuf specifications for messages used within Sigstore.
+* Language clients for different ecosystems.
+
+During a release, a few steps have to be synchronized to release the
+messages and the language clients.
+
+## Notes on semantic versioning
+
+### Major version change
+As expected this indicate a breaking change. Any major update MUST
```

Comment:
Maybe a couple of examples of breaking changes? Field deletions or renames, tag number changes, type change